README.md

```diff
@@ -118,101 +118,87 @@ Who Uses Nomad
+    $ cd nomad
+    $ vagrant up
+    ```
 
 The virtual machine will launch, and a provisioning script will install the
-needed dependencies.
-
-**Developing locally**
-For local dev first make sure Go is properly installed, including setting up a
-[GOPATH](https://golang.org/doc/code.html#GOPATH). After setting up Go, clone this
-repository into `$GOPATH/src/github.com/hashicorp/nomad`. Then you can
-download the required build tools such as vet, cover, godep etc by bootstrapping
-your environment.
-
-```sh
-$ make bootstrap
-...
-```
-
-Nomad creates many file handles for communicating with tasks, log handlers, etc.
-In some development environments, particularly macOS, the default number of file
-descriptors is too small to run Nomad's test suite. You should set
-`ulimit -n 1024` or higher in your shell. This setting is scoped to your current
-shell and doesn't affect other running shells or future shells.
-
-Afterwards type `make test`. This will run the tests. If this exits with exit status 0,
-then everything is working!
-
-```sh
-$ make test
-...
-```
-
-To compile a development version of Nomad, run `make dev`. This will put the
-Nomad binary in the `bin` and `$GOPATH/bin` folders:
-
-```sh
-$ make dev
-```
-
-Optionally run Consul to enable service discovery and health checks:
-
-```sh
-$ sudo consul agent -dev
-```
-
-And finally start the nomad agent:
-
-```sh
-$ sudo bin/nomad agent -dev
-```
-
-If the Nomad UI is desired in the development version, run `make dev-ui`. This will build the UI from source and compile it into the dev binary.
-
-```sh
-$ make dev-ui
-...
-$ bin/nomad
-...
-
-To compile protobuf files, installing [buf](https://docs.buf.build/installation) is required.
-
-```
-
-**Note:** Building the Nomad UI from source requires Node, Yarn, and Ember CLI. These tools are already in the Vagrant VM. Read the [UI README](https://github.com/hashicorp/nomad/blob/master/ui/README.md) for more info.
-
-To cross-compile Nomad, run `make prerelease` and `make release`.
-This will generate all the static assets, compile Nomad for multiple
-platforms and place the resulting binaries into the `./pkg` directory:
+needed dependencies within the VM.
+
+### Developing without Vagrant
+1. Install [Go 1.15.5+](https://golang.org/) *(Note: `gcc-go` is not supported)*
+1. Clone this repo
+   ```sh
+   $ git clone https://github.com/hashicorp/nomad.git
+   $ cd nomad
+   ```
+1. Bootstrap your environment
+   ```sh
+   $ make bootstrap
+   ```
+1. (Optionally) Set a higher ulimit, as Nomad creates many file handles during normal operations
+   ```sh
+   $ [ "$(ulimit -n)" -lt 1024 ] && ulimit -n 1024
+   ```
+1. Verify you can run tests
+   ```sh
+   $ make test
+   ```
+
+### Running a development build
+
+1. Compile a development binary (see the [UI README](https://github.com/hashicorp/nomad/blob/master/ui/README.md) to include the web UI in the binary)
+    ```sh
+    $ make dev
+    # find the built binary at ./bin/nomad
```

Comment:
It looks like this path is coming from the previous version of the directions, but I think it ends up at `~/go/bin/nomad` by default?

Reply:
It gets installed to multiple places by default:
* `./bin/nomad`
* `./pkg/$GOOS_$GOARCH/nomad`
* `$GOPATH/bin/nomad`

See https://github.com/hashicorp/nomad/blob/master/GNUmakefile#L288.

There's no real need for it to be installed into `$GOPATH/bin`, except that people may be depending on `$GOPATH/bin` having been added to their `$PATH` as a way to run a dev build.

Reply:
Oh? (Rummages around on file system) 🤦 sure does, doesn't it!